src/main/java/com/epam/lowcost/service/implementations/FlightServiceImpl.java

```diff
@@ -9,14 +9,22 @@
 import java.time.LocalDateTime;
 import java.util.List;
 
+import static java.time.temporal.ChronoUnit.DAYS;
+
+
 @AllArgsConstructor
 public class FlightServiceImpl implements FlightService {
     private FlightDAO flightDAO;
     private PlaneService planeService;
 
     @Override
     public List<Flight> getAllFlights() {
-        return flightDAO.getAllFlights();
+        List<Flight> flights = flightDAO.getAllFlights();
```

Comment:
I think we should separate "getting all filghts" and "getting all flights with updated price"